src/sql/engine/px/ob_dfo_scheduler.cpp

```diff
@@ -463,6 +463,14 @@ int ObSerialDfoScheduler::dispatch_sqcs(ObExecContext &exec_ctx,
                               coord_info_.interrupt_id_,
                               &sqc);
         args.set_serialize_param(exec_ctx, const_cast<ObOpSpec &>(*dfo.get_root_op_spec()), *phy_plan);
+        // 设置sqc的record_data_字段
+        ObQcRecordData& record_data = sqc.get_record_data();
+        record_data.tid = getpid();
```

Comment:
getpid() returns thread id or process id? what do we need here?

Reply:
and, How we define tid? It is the px coordinator's thread id? or the main sql thread's thread id?

as we have nested px, which is executed in a different thread to main sql's. we need to define this explicitly.

Reply:
> and, How we define tid? It is the px coordinator's thread id? or the main sql thread's thread id?
> 
> as we have nested px, which is executed in a different thread to main sql's. we need to define this explicitly.

1. It's thread id, i made a mistake
2. It is the px coordinator's thread id, and i'll leave a comment in the struct～